metaflow/plugins/kfp/kfp.py

```diff
@@ -209,7 +211,9 @@ def copy_log_cmd(log_file):
         #  where the ordinal is attempt/retry count
         cp_stderr = copy_log_cmd(log_file="0.stderr.log")
         cp_stdout = copy_log_cmd(log_file="0.stdout.log")
-        cp_logs_cmd = f"{cp_stderr} && {cp_stdout}"
+        cp_logs_cmd = (
+            "set -x" if debug.subcommand else "true" f" && {cp_stderr} && {cp_stdout}"
```

Comment:
Could you please elaborate on why set -x solves the problem?

Reply:
This doesn't solve the problem, it is only for debugging when `export METAFLOW_DEBUG_SUBCOMMAND=1`

Reply:
Is this intended to be merged then?